apps/judicial-system/web/src/components/InfoCard/useInfoCardItems.tsx

```
@@ -36,42 +35,55 @@ const useInfoCardItems = () => {
+        </Text>
+      ),
+      values: defendants
+        ? defendants.map((defendant, index) => (
+            <div
+              className={cn(
+                isMultipleDefendants ? styles.renderDivider : undefined,
+                defendants && index === defendants.length - 1
+                  ? styles.last
+                  : undefined,
+              )}
+            >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)